docs-requirements.txt

```diff
@@ -9,7 +9,9 @@ idna==2.10
 imagesize==1.2.0
 incremental==17.5.0
 jinja2==2.11.2
+git+git://github.com/inspirehep/jsonschema2rst@d211d9709046431a6b6a4594c97c22d8713d854b#egg=jsonschema2rst
```

Comment:
Why this specific commit? I guess there's no package in pypi?

Reply:
The package maintainer does not adhere to a convention versioning scheme on PyPI, so this is the only way to reference the previous version (current one has a bug) .

Reply:
New version (0.1.4) includes too old pytest in normal installation, old version (0.1.0) has issues with metadata